drivers/source/PwmOut.cpp

```diff
@@ -98,6 +98,26 @@ void PwmOut::pulsewidth_us(int us)
     core_util_critical_section_exit();
 }
 
+void PwmOut::suspend()
+{
+    core_util_critical_section_enter();
+    if (_initialized) {
+        _duty_cycle = PwmOut::read();
+        PwmOut::deinit();
+    }
+    core_util_critical_section_exit();
+}
+
+void PwmOut::resume()
+{
+    core_util_critical_section_exit();
```

Comment:
Should be core_util_critical_section_enter()

Reply:
Corrected [here](https://github.com/ARMmbed/mbed-os/compare/1645cc774b8a27fad92693314e16001133578e25..fdc76051f30588bac2df8155c26ba6cd700b5879).
Thanks.